build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")"
```

Comment:
I'd prefer this script live in `hack/build.sh` (because it's a script for hacking on this project), in which case this would be `cd "$(dirname "$0")/.."`